tests/test_ast_formatter.py

```
@@ -0,0 +1,124 @@
+        assert text.startswith("ProgramNode(")
+        assert "NumberNode(" in text
+        assert "value=5" in text
+
+    def test_multiline_indentation(self):
+        text = plain(ProgramNode([NumberNode(5)]))
+        assert "\n" in text
+        # the statements list child is indented deeper than the root
+        lines = text.split("\n")
+        root = lines[0]
+        assert not root.startswith("    ")
+        assert any(l.startswith("    statements=[") for l in lines)
```

Ruff (0.16.2)

[error] Ambiguous variable name: `l`

(E741)